xml/containers-get.xml

```diff
@@ -49,44 +48,26 @@
    <title>Repository Names</title>
    <para>
     Repository names start with the name of the product, for example:
-    <literal>suse/sle/(…​)</literal>,
-    <literal>opensuse/tumbleweed/(…​)</literal>,
-    <literal>caasp/(…​)</literal>. The &slea; 15 containers for all
-    service packs reside in the namespace <literal>suse/sle15</literal>. This
+    <literal>suse/sle</literal>,
+    <literal>opensuse/tumbleweed</literal>,
+    <literal>caasp</literal>. The &slea; 15 containers for all
+    service packs reside in the repository <literal>suse/sle15</literal>. This
     applies to future versions of &slea; as well (for example,
     <literal>suse/sle16</literal>). However, for &slea; 12, there is a separate
     repository name for each service pack, for example
-    <literal>suse/sle12sp3</literal>, <literal>suse/sle12sp4</literal>,
-    <literal>suse/sle12sp5</literal>.
+    <literal>suse/sles12sp3</literal>, <literal>suse/sles12sp4</literal>,
+    <literal>suse/sles12sp5</literal>.
    </para>
   </sect2>
 
   <sect2 xml:id="sec-labels">
    <title>Labels</title>
    <para>
-    Labels help to identify images. All &slea; container image labels begin
-    with <literal>com.suse.&lt;product&gt;.&lt;containername&gt;</literal>
-    after which follows a further specification. &opensuse; container images
-    are identified by a label that begins with
-    <literal>org.opencontainers</literal>.
+    Labels help to identify images. All &slea; container image labels begin with
+    <literal>com.suse.<replaceable>PRODUCT</replaceable><replaceable>CONTAINER_NAME</replaceable></literal>
+    followed by a further specification. Container images may also contain
+    <literal>org.opencontainers.image</literal> labels.
    </para>
-   <important>
-    <title>The <literal>com.suse.sle.base</literal> label is protected.</title>
```

Comment:
There should be some replacement of this section which explains the `org.opencontainers.image` labels like https://en.opensuse.org/Building_derived_containers#Labels

Reply:
Added to my to-do.